_posts/2016-04-05-nectar-overview.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+cover: 'assets/images/general-cover-3.jpg'
+title: NectarCommerce Overview
+tags: docs
+subclass: 'post tag-docs'
+categories: 'elixir'
+author: 'Pikender'
+navigation: true
+logo: 'assets/images/nectar-cart.png'
+---
+
+Vision
+======
+
+**E-commerce Framework with an extension methodology which allows easy integration of custom features unique to the domain.**
+
+## NectarCommerce in NutShell
+
+[NectarCommerce](https://github.com/vinsol/nectarcommerce) is an Elixir/Phoenix Application which is currently having following E-commerce features:
```

Comment:
Probably we should mention that it is an e-commerce elixir/phoenix application instead of saying a phoenix application with e-commerce features.